docs/modules/position.md

```diff
@@ -0,0 +1,609 @@
+
+Example
+
+:   Get your position relative to the nearest Creeper (without scaling):
+
+    ```
+    # Once
+    execute as @s at @e[type=creeper,limit=1,sort=nearest] run function #bs.position:get_relative_ata {scale:1}
+
+    # See the result
+    tellraw @a [{"text": "Relative position : X=", "color": "dark_gray"},{"score":{"name":"$position.get_relative_ata.x", "objective": "bs.out"}, "color": "gold"},{"text":", Y=", "color": "dark_gray"},{"score":{"name":"$position.get_relative_ata.y", "objective": "bs.out"},"color":"gold"},{"text":", Z=","color":"dark_gray"},{"score":{"name":"$position.get_relative_ata.z","objective":"bs.out"},"color":"gold"}]
+    ```
+
+> **Credits**: Aksiome, Leirof
+
+---
+
+### Get distance
+
+::::{tab-set}
+:::{tab-item} Normal
+
+**`#bs.position:get_distance_ata {scale:<scaling>}`**
+
+Calculates the distance between the source entity and the execution position of the function.
+
+Inputs
+
+:   (execution) `as <entities>`
+    : The entity positioned at the reference point.
+
+    (execution) `at <entity>` or `positioned <x> <y> <z>`
+    : The position to get the distance from.
+
+    (macro variable) `scale`: double
+    : Scalar for the function's outputs.
+
+Outputs
+
+:   (score) `$position.get_distance_ata bs.out`
+    : The distance between the two positions.
+
+Example
+
+:   Calculate the distance between you and the nearest sheep (in blocks, without scaling):
+
+    ```
+    # Once
+    execute at @e[type=sheep,limit=1,sort=nearest] run function #bs.position:get_distance_ata {scale:1}
+
+    # See the result
+    tellraw @a [{"text": "Distance: ", "color": "dark_gray"},{"score":{"name":"$position.get_distance_ata", "objective": "bs.out"}, "color": "gold"}]
+    ```
+
+```{admonition} Performance tip
+:class: tip
+
+For several applications such as comparing distances, you can use the squared distance instead of the "normal" distance. It is a bit faster to calculate.
+```
+
+:::
+:::{tab-item} Squared
+
+**`#bs.position:get_distance_squared_ata {scale:<scaling>}`**
+
+Calculates the squared distance between the source entity and the execution position of the function.
+
+Inputs
+
+:   (execution) `as <entities>`
+    : The entity positioned at the reference point.
+
+    (execution) `at <entity>` or `positioned <x> <y> <z>`
+    : The position to get the distance from.
+
+    (macro variable) `scale`: double
+    : Scalar for the function's outputs.
+
+Outputs
+
+:   (score) `$position.get_distance_squared_ata bs.out`
+    : The squared distance between the two positions.
+
+Example
+
+:   Calculate the squared distance between you and the nearest sheep (in blocks, without scaling):
+
+    ```
+    # Once
+    execute at @e[type=sheep,limit=1,sort=nearest] run function #bs.position:get_distance_squared_ata {scale:1}
+
+    # See the result
+    tellraw @a [{"text": "Distance^2: ", "color": "dark_gray"},{"score":{"name":"$position.get_distance_squared_ata", "objective": "bs.out"}, "color": "gold"}]
+    ```
```

Comment:
Output aberrant values:
```
/execute positioned ~1 ~2 ~3 run function #bs.position:get_distance_squared_ata {scale:1}
```
Gives 10037 instead of 14

Reply:
Huh weird, just tested it and got 14